I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 42, in script
    $ player_name = player_name[0].upper() + player_name[1:]
  File "game/script.rpy", line 42, in <module>
    $ player_name = player_name[0].upper() + player_name[1:]
IndexError: string index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 42, in script
    $ player_name = player_name[0].upper() + player_name[1:]
  File "C:\Users\sanja\Documents\renpy-7.4.11-sdk/renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\sanja\Documents\renpy-7.4.11-sdk/renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 42, in <module>
    $ player_name = player_name[0].upper() + player_name[1:]
IndexError: string index out of range

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
HeartInCode 1.0
Sat Feb 12 00:03:03 2022
